llama_stack/providers/tests/memory/conftest.py

```diff
@@ -74,6 +69,19 @@ def pytest_configure(config):
 
 
 def pytest_generate_tests(metafunc):
+    test_config = try_load_config_file_cached(metafunc.config.getoption("config"))
```

Comment:
seems like this block of code (upto `custom_fixtures`) can all be moved into the utility method?